BuildResidentialHPXML/measure.rb

```diff
@@ -2963,169 +2963,113 @@ def validate_arguments(args)
     # TODO: Remove items below that duplicate checks downstream.
     # TODO: Add warnings/errors downstream if there are additional checks here.
 
-    # heat pump water heater with natural gas fuel type
     warning = ([HPXML::WaterHeaterTypeHeatPump].include?(args[:water_heater_type]) && (args[:water_heater_fuel_type] != HPXML::FuelTypeElectricity))
-    warnings << "water_heater_type=#{args[:water_heater_type]} and water_heater_fuel_type=#{args[:water_heater_fuel_type]}" if warning
+    warnings << 'Cannot model a heat pump water heater with non-electric fuel type.' if warning
 
-    # heating system and heat pump
     error = (args[:heating_system_type] != 'none') && (args[:heat_pump_type] != 'none') && (args[:heating_system_fraction_heat_load_served] > 0) && (args[:heat_pump_fraction_heat_load_served] > 0)
-    errors << "heating_system_type=#{args[:heating_system_type]} and heat_pump_type=#{args[:heat_pump_type]}" if error
+    errors << 'Multiple central heating systems are not currently supported.' if error
 
-    # cooling system and heat pump
     error = (args[:cooling_system_type] != 'none') && (args[:heat_pump_type] != 'none') && (args[:cooling_system_fraction_cool_load_served] > 0) && (args[:heat_pump_fraction_cool_load_served] > 0)
-    errors << "cooling_system_type=#{args[:cooling_system_type]} and heat_pump_type=#{args[:heat_pump_type]}" if error
+    errors << 'Multiple central cooling systems are not currently supported.' if error
 
-    # non integer number of bathrooms
     if args[:geometry_unit_num_bathrooms] != Constants.Auto
       error = (Float(args[:geometry_unit_num_bathrooms]) % 1 != 0)
-      errors << "geometry_unit_num_bathrooms=#{args[:geometry_unit_num_bathrooms]}" if error
+      errors << 'Currently only supporting an integer number of bathrooms.' if error
     end
 
-    # non integer ceiling fan quantity
     if args[:ceiling_fan_quantity] != Constants.Auto
       error = (Float(args[:ceiling_fan_quantity]) % 1 != 0)
-      errors << "ceiling_fan_quantity=#{args[:ceiling_fan_quantity]}" if error
+      errors << 'Currently only supporting an integer quantity of ceiling fans.' if error
     end
 
-    # single-family, slab, foundation height > 0
-    warning = [HPXML::ResidentialTypeSFD, HPXML::ResidentialTypeSFA].include?(args[:geometry_unit_type]) && (args[:geometry_foundation_type] == HPXML::FoundationTypeSlab) && (args[:geometry_foundation_height] > 0)
-    warnings << "geometry_unit_type=#{args[:geometry_unit_type]} and geometry_foundation_type=#{args[:geometry_foundation_type]} and geometry_foundation_height=#{args[:geometry_foundation_height]}" if warning
+    warning = (args[:geometry_foundation_type] == HPXML::FoundationTypeSlab) && (args[:geometry_foundation_height] > 0)
+    warnings << 'Specified slab or above apartment foundation type with a non-zero foundation height. Assuming foundation height is zero.' if warning
```

Comment:
The warning refers to both slab and above apartment foundation types, but the line above only references a slab foundation type. Seems like an inconsistency?